src/ThirdParty/ThirdParty.router.ts

```diff
@@ -32,6 +41,7 @@ export class ThirdPartyRouter extends Router {
      */
     this.router.get(
       '/thirdParties/:id',
+      withCors,
```

Comment:
should we add: `this.router.options('/thirdParties/:id', withCors)`?

Reply:
Good catch, added!